src/ui/TemplateMessageItemBody/index.tsx

```diff
@@ -38,6 +38,27 @@ const getFilledMessageTemplateWithData = (
   return parsedTemplate;
 };
 
+const parseTemplateWithReplaceReplacer = (
+  templateString: string,
+  templateVariables: Record<string, string>,
+  colorVariables: Record<string, unknown>,
+  theme: SendbirdTheme,
+): MessageTemplateItem[] => {
```

Comment:
better to use object type?

Reply:
I removed this since not used right now. will add again in the future